components/opponent/commons/opponent_display.lua

```diff
@@ -469,10 +469,10 @@ end
 ---@return string
 function OpponentDisplay.InlineScore(opponent)
 	if opponent.status == 'S' then
```

Comment:
S Status should not have a nil score, so let's solve this at the root cause instead